src/soc/ibm/power9/chip.c

```diff
@@ -183,16 +146,22 @@ static void activate_slave_cores(void)
 
 void platform_prog_run(struct prog *prog)
 {
-	clear_ipmi_attn();
+	enum { IPMI_BT_PORT_BASE = 0xE4 };
 
 	/*
-	 * Now that the payload and its interrupt vectors are already loaded
-	 * perform 16.2.
+	 * Clear SMS_ATN aka EVT_ATN in BT_CTRL - Block Transfer IPMI protocol
+	 *
+	 * BMC sends event telling us that HIOMAP (access to flash, either real or
+	 * emulated, through LPC) daemon has been started. This sets the mentioned bit.
+	 * Skiboot enables interrupts, but because those are triggered on 0->1
+	 * transition and bit is already set, they do not arrive.
 	 *
-	 * This MUST be done as late as possible so that none of the newly
-	 * activated threads start execution before current thread jumps into
-	 * the payload.
```

Comment:
Please leave this comment immediately above activate_slave_cores() so nobody tries to add code after it.